pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "klaam"
+version = "0.0.1"
+description = "Arabic speech recognition, classification and text-to-speech."
+license = "MIT"
+authors = [
+    "Zaid Alyafeai <alyafey22@gmail.com>",
+    "mustafa0x"
+]
+readme = 'README.md'
+
+[tool.poetry.dependencies]
+python = "^3.8"
+rich = "^12.2.0"
+jupyter = "^1.0.0"
+jiwer = "^2.3.0"
+arabic-pronounce = "^0.2.5"
+mishkal = "^0.4.1"
+codernitydb3 = "^0.6.0"
+pypinyin = "^0.46.0"
+transformers = "^4.18.0"
+datasets = "^2.1.0"
+Unidecode = "^1.3.4"
+inflect = "^5.5.2"
+
+[tool.poetry.dev-dependencies]
+pytest = "^7.1.1"
+pylint = "^2.13.5"
+autopep8 = "^1.6.0"
+gdown = "^4.4.0"
+pandas = "1.3.5"
```

Comment:
I think pandas,tqdm should be in dependencies instead of dev-dependencies.

Reply:
I agree with you. I will move it to `dependencies` rather than `dev-dependencies`.